packages/@coorpacademy-components/src/organism/list-item/index.js

```diff
@@ -17,9 +19,18 @@ const ListItem = props => {
   })(tags);
   return (
     <div className={style.wrapper}>
-      <div className={style.title} title={title}>
-        {title}
-      </div>
+      {isPublished ? (
+        <div className={style.orderWrapper}>
+          <div className={style.order}>{order + 1}</div>
```

Comment:
you need to add an aria-label to explain the meaning of this number ( for your cas it will be like "The order of the certification" )